bank.engineering.yaml

```diff
@@ -3,3 +3,16 @@ changelog:
   ttl: 1
   type: CNAME
   value: headwayapp.co.
+_acme-challenge.curl:
+  ttl: 1
+  type: CNAME
+  value: c82b414cd89d32fd90685ead._acme.deno.dev.
+curl:
+  ttl: 1
+  type: A
+  value: 34.120.54.55
+curl:
+  ttl: 1
+  type: AAAA
+  value: 2600:1901:0:6d85::

```

Comment:
Wrap this in quotes since it contains colons

Reply:
ok